lgsm/config-default/config-lgsm/armareforgerserver/_default.cfg

```diff
@@ -0,0 +1,183 @@
+##################################
+######## Default Settings ########
+##################################
+# DO NOT EDIT, ANY CHANGES WILL BE OVERWRITTEN!
+# Copy settings from here and use them in either:
+# common.cfg - applies settings to every instance.
+# [instance].cfg - applies settings to a specific instance.
+
+#### Game Server Settings ####
+
+## Predefined Parameters | https://docs.linuxgsm.com/configuration/start-parameters
+ip="0.0.0.0"
+
+## Server Parameters | https://docs.linuxgsm.com/configuration/start-parameters#additional-parameters
+startparameters="-config ${servercfgfullpath} -backendlog -nothrow -profile ${serverprofile} -gproj ${gproj}" 
```

Comment:
Looks like this is taken from the manual:
https://community.bistudio.com/wiki/Arma_Reforger:Server_Hosting
Do we really need here all ?